bitpacker/src/bitpacker.rs

```diff
@@ -118,6 +122,125 @@ impl BitUnpacker {
         let val_shifted = val_unshifted_unmasked >> bit_shift;
         val_shifted & self.mask
     }
+
+    // Decodes the range of bitpacked `u32` values with idx
+    // in [start_idx, start_idx + output.len()).
+    //
+    // #Panics
+    //
+    // This methods panics if `num_bits` is > 32.
+    fn get_batch_u32s(&self, start_idx: u32, data: &[u8], output: &mut [u32]) {
```

Comment:
I made it private upon your suggestion. We can make it public if it because useful.